react/graphql/AddProductTranslations.graphql

```diff
@@ -1,3 +1,5 @@
 mutation AddProductTranslations($translations: [MessagesFromProvider]!, $language: String!) {
-  addProductTranslations(translations: $translations, language: $language)
+  addProductTranslations(translations: $translations, language: $language) {
+    provider
```

Comment:
I'd consider adding a comment here saying that these are the translations that failed